src/common/TrackedOp.cc

```diff
@@ -214,7 +214,11 @@ bool OpTracker::check_ops_in_flight(std::vector<string> &warning_vector)
   //store summary message
   warning_vector.push_back("");
 
-  int slow = 0;     // total slow
+  int _slow = 0;    // total slow
+  if (!slow)
+    slow = &_slow; 
+  else
+    *slow = _slow;  // start from 0 anyway
```

Comment:
@ukernel  yes, you're right, but I chose 'slow' as a input/output parameter to avoid overloading check_ops_in_flight. Now I change it since you ask. Pls review the latest changes. Thanks.